bundle/manifests/external-secrets-operator.clusterserviceversion.yaml

```diff
@@ -32,6 +32,8 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "true"
     features.operators.openshift.io/token-auth-azure: "true"
     features.operators.openshift.io/token-auth-gcp: "true"
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.19"}, {"type":
+      "olm.minOpenShiftVersion", "value": "4.19"}]'
```

Comment:
we need to remove the olm.minOpenShiftVersion,
rest looking good per se clarifications in https://redhat-internal.slack.com/archives/C3VS0LV41/p1747939698126179?thread_ts=1747915033.683459&cid=C3VS0LV41

/lgtm cancel
/label px-approved